analytics/trend_analyzer.py

```
@@ -0,0 +1,631 @@
+        # 分析結果の集計
+        for result in analysis_results:
+            date_key = result.created_at.date().isoformat()
+            daily_stats[date_key]['analyses'] += 1
+            
+            if result.analysis_result:
+                scores = result.analysis_result.get('scores', {})
+                for skill, score in scores.items():
+                    daily_stats[date_key]['avg_scores'][skill].append(score)
+        
+        # 平均スコアの計算
+        for date_key, stats in daily_stats.items():
```

Ruff (0.12.2)

Loop control variable `date_key` not used within loop body

(B007)